src/main/resources/io/viash/runners/nextflow/VDSL3Helper.nf

```diff
@@ -259,10 +263,10 @@ def _vdsl3ProcessFactory(Map workflowArgs, Map meta, String rawScript) {
   |$stub
   |\"\"\"
   |script:$assertStr
-  |def escapeText = { s -> s.toString().replaceAll('([`"])', '\\\\\\\\\$1') }
+  |def escapeText = { s -> s.toString().replaceAll("'", "'\\\"'\\\"'") }
```

Comment:
We could probably move this function above the `process $procKey` statement. if it works, that would ensure that it's clear that this is available to the nextflow process to use throughout.

Reply:
done